interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -130,7 +130,7 @@ protected static void startStaticServer(
         .build();
 
     builder.addService(ServerInterceptors.intercept(
-        TestServiceGrpc.bindService(new TestServiceImpl(testServiceExecutor)),
+        new TestServiceImpl(testServiceExecutor).bindService(),
```

Comment:
Oh, it looks like we need a `intercept(BindableService, List<ServerInterceptor>)` overload. Please add that, since users will be reasonably prone to needing it.